src/reactjs_jsx_ppx_v4.ml

```diff
@@ -0,0 +1,1455 @@
+("icon", {jsString = None; typeString = mkPropConst (Lident "string")});
+("inputMode", {jsString = None; typeString = mkPropConst (Lident "string")});
+("integrity", {jsString = None; typeString = mkPropConst (Lident "string")});
+("keyType", {jsString = None; typeString = mkPropConst (Lident "string")});
+("kind", {jsString = None; typeString = mkPropConst (Lident "string")});
+("label", {jsString = None; typeString = mkPropConst (Lident "string")});
+("list", {jsString = None; typeString = mkPropConst (Lident "string")});
+("loop", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("low", {jsString = None; typeString = mkPropConst (Lident "int")});
+("manifest", {jsString = None; typeString = mkPropConst (Lident "string")});
+("max", {jsString = None; typeString = mkPropConst (Lident "string")});
+("maxLength", {jsString = None; typeString = mkPropConst (Lident "int")});
+("media", {jsString = None; typeString = mkPropConst (Lident "string")});
+("mediaGroup", {jsString = None; typeString = mkPropConst (Lident "string")});
+("method", {jsString = None; typeString = mkPropConst (Lident "string")});
+("min", {jsString = None; typeString = mkPropConst (Lident "string")});
+("minLength", {jsString = None; typeString = mkPropConst (Lident "int")});
+("multiple", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("muted", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("name", {jsString = None; typeString = mkPropConst (Lident "string")});
+("nonce", {jsString = None; typeString = mkPropConst (Lident "string")});
+("noValidate", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("open_", {jsString = Some("open"); typeString = mkPropConst (Lident "bool")});
+("optimum", {jsString = None; typeString = mkPropConst (Lident "int")});
+("pattern", {jsString = None; typeString = mkPropConst (Lident "string")});
+("placeholder", {jsString = None; typeString = mkPropConst (Lident "string")});
+("poster", {jsString = None; typeString = mkPropConst (Lident "string")});
+("preload", {jsString = None; typeString = mkPropConst (Lident "string")});
+("radioGroup", {jsString = None; typeString = mkPropConst (Lident "string")});
+("readOnly", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("rel", {jsString = None; typeString = mkPropConst (Lident "string")});
+("required", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("reversed", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("rows", {jsString = None; typeString = mkPropConst (Lident "int")});
+("rowSpan", {jsString = None; typeString = mkPropConst (Lident "int")});
+("sandbox", {jsString = None; typeString = mkPropConst (Lident "string")});
+("scope", {jsString = None; typeString = mkPropConst (Lident "string")});
+("scoped", {jsString = None; typeString = mkPropConst (Lident "bool")});
+("scrolling", {jsString = None; typeString = mkPropConst (Lident "string")});
```

Comment:
According to [HTML 4.01](https://www.w3.org/TR/html401/present/frames.html#adef-scrolling), this can assume only fixed values. Note that this is an HTML 4 only attribute.
```suggestion
("scrolling", {jsString = None; typeString = mkPropVariant ["auto"; "yes"; "no"]});
```